pipenv/cli/options.py

```diff
@@ -156,6 +157,17 @@ def keep_outdated_option(f):
     def callback(ctx, param, value):
         state = ctx.ensure_object(State)
         state.installstate.keep_outdated = value
+        if value:
+            click.echo(
+                click.style(
+                    "Use of --keep-outdated has been deprecated for removal."
+                    "The flag does not respect package resolver results and leads to inconsistent lock files.  "
```

Comment:
I prefer
"The flag --keep-outdated has been deprecated and marked for removal.
It does not respect ..."